ctapipe/core/tool.py

```diff
@@ -189,18 +206,38 @@ def initialize(self, argv=None):
         self.update_logging_config()
 
         if self.config_file is not None:
-            self.log.debug(f"Loading config from '{self.config_file}'")
+            self.log.info(f"Loading config from '{self.config_file}'")
```

Comment:
Move into loop

Reply:
`Provenance().add_input(config_file, "config")`?

Reply:
good point! I'm actually surprised it wasn't there before